docs/concepts/stateful-stream-processing.md

```diff
@@ -123,7 +123,7 @@ provided by Flink's connectors.
 snapshots, we use the words *snapshot* and *checkpoint* interchangeably. Often
 we also use the term *snapshot* to mean either *checkpoint* or *savepoint*.
 
-### Checkpointing
+### Aligned Checkpointing
```

Comment:
Hum, not sure I'd change this title, it might be more confusing than helpful. The sentence you have below is clarifying enough ("Since Flink 1.11, checkpoints can be taken with or without alignment. In the 
following, we describe aligned checkpoints first.").